lib/git_fame/base.rb

```diff
@@ -261,12 +264,19 @@ def convert_exclude_paths_to_array
     end
 
     #
+    # Converts @extensions argument to an array
+    #
+    def convert_extensions_to_array
+      @extensions.split(",")
+    end
+
+    #
     # Removes files matching paths in @exclude from @files instance variable
     #
     def remove_excluded_files
       return if @exclude.empty?
       @files = @files.map do |path|
-        next if  path =~ /\A(#{@exclude.join("|")})/
+        next if path =~ /\A(#{@exclude.join("|")})/
         path
       end.compact
```

Comment:
Aha, true. I thought it was your code. I'll review your PR later this week. Thanks!